addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -153,7 +159,7 @@ export class EmojiPicker extends Component {
             this.emojiByCodepoints = Object.fromEntries(
                 this.emojis.map((emoji) => [emoji.codepoints, emoji])
             );
-            this.state.categoryId = this.categories[0].sortId;
+            this.state.categoryId = this.categories?.[0]?.sortId;
```

Comment:
I'd rather initializing categories/emojis to an empty array, easier to work with, no more `?.[0]` required.